drivers/hmis/app/controllers/hmis/base_controller.rb

```diff
@@ -8,7 +8,6 @@ class Hmis::BaseController < ApplicationController
   include Hmis::Concerns::JsonErrors
   respond_to :json
   before_action :set_csrf_cookie
-  protect_from_forgery with: :reset_session
```

Comment:
There's no point in setting this because we are overriding `handle_unverified_request` anyway